debian/bin/build

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env bash
 
 electron-packager . \
-  --ignore=node_modules/electron-prebuilt \
-  --ignore=node_modules/electron-packager \
```

Comment:
are you sure this needs to be removed? this will make the `.deb` file twice as big